source/server/lds_subscription.h

```diff
@@ -51,6 +52,7 @@ class LdsSubscription : public Http::RestApiFetcher,
   const LocalInfo::LocalInfo& local_info_;
   Config::SubscriptionCallbacks<envoy::api::v2::Listener>* callbacks_ = nullptr;
   Config::SubscriptionStats stats_;
+  const Stats::Scope& scope_;
```

Comment:
similar comment here about scope (and options), maybe applies elsewhere also.

Reply:
Agreed -- see https://github.com/envoyproxy/envoy/pull/3629#discussion_r201387197.